trolo/train.py

```diff
@@ -110,7 +109,7 @@ def train_model(
         if "HGNetv2" in cfg.yaml_cfg:
             cfg.yaml_cfg["HGNetv2"]["pretrained"] = False
 
-    print("cfg: ", cfg.__dict__)
+    LOGGER.info(f"cfg: {cfg.__dict__}")
```

Comment:
can remove this log, it was meant for debugging

Reply:
Done